README.md

```diff
@@ -14,10 +14,44 @@ Welcome to NI's internal and external C# conventions and linter rules/plugins/to
 
 ## Quickstart
 
-Add a package reference to the package TODO_TBD.
+### Introduction
+Add a package reference to the package `NI.CSharp.Analyzers`.
 
 ```msbuild
-<PackageReference Include="" Version="" />
+<PackageReference Include="NI.CSharp.Analyzers" Version="0.1.0" />
```

Comment:
this should be adjusted if we want to host the package in GitHub.

Reply:
Yeah, you are right. I completely missed the step of "adding a reference to the package source". I will add that and keep the actual source <to-do>.